assets/js/vue/components/search/Facet.vue

```diff
@@ -10,7 +10,7 @@
                 <div class="input-group pb-3" v-show="['researchGroup', 'projectDirector'].includes(facetName.queryParam)">
                     <input class="form-control" placeholder="Search" type="text" v-model="facetSearch">
                     <div class="input-group-append">
-                        <button class="btn btn-primary" type="button">
+                        <button class="btn btn-alternate" type="button">
```

Comment:
This will also change the color for the other templates, are "we" ok with that?

Reply:
It appears they are the same colors, so should be ok.

Reply:
Yes, it should be because for other templates we have primary and alternate as the same color